app/components/util/ranking.jsx

```diff
@@ -35,7 +35,6 @@ export default function Ranking(props) {
       )
     } else onDone({ valid: true, value: e.target.value })
   }
```

Comment:
There doen't seem to be a change to this file, except formatting. If that's the case, please revert it back to the original.